### PR TITLE
164 make leave neighborhood button functional

### PR DESCRIPTION
**Pull request by radu-constantin**
Leave Neighborhood button opens up an alert asking the user if he/she is sure before leaving the neighborhood and redirecting to the home page.

### CODE REVIEW

File: apps/frontend/src/components/DescriptionBox/DescriptionBox.module.css [hunk 1]
```diff
@@ -40,6 +40,17 @@
   margin: 10px;
 }
 
+.alertBtnContainer {
+  margin: 0 auto;
```
Comment (nina-dk) [inline apps/frontend/src/components/DescriptionBox/DescriptionBox.module.css:44]: Could we give some more top margin?

File: apps/frontend/src/components/ResponseBox/ResponseBox.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ export default function ResponseBox({ response, requestOwnerId }: Props) {
           text="Accept offer"
         />
       );
-    } else if (responseOwner) {
+    }  if (responseOwner) {
```
Comment (nina-dk) [inline apps/frontend/src/components/ResponseBox/ResponseBox.tsx:56]: this should probably be on a new line

**Review by nina-dk — APPROVED** [review]
I noticed we currently don't display the "leave" button to the neighborhood admin. If it's going to remain that way, we can do away with the special handling of deleting the neighborhood etc at the backend. Otherwise, we need to change this on the frontend. Additionally, when someone leaves a neighborhood and then joins back, do we want their requests to be revived?

Great job, as usual!

File: apps/frontend/src/components/DescriptionBox/DescriptionBox.module.css (hunk 1 above)
Comment (radu-constantin) [reply apps/frontend/src/components/DescriptionBox/DescriptionBox.module.css:44]: Done! Let me know if it's enough!

File: apps/frontend/src/components/ResponseBox/ResponseBox.tsx (hunk 2 above)
Comment (radu-constantin) [reply apps/frontend/src/components/ResponseBox/ResponseBox.tsx]: Done!

**Pull request merged**